poetry.lock

```diff
@@ -504,6 +504,68 @@ files = [
     {file = "pytz-2024.1.tar.gz", hash = "sha256:2a29735ea9c18baf14b448846bde5a48030ed267578472d8955cd0e7443a9812"},
 ]
 
+[[package]]
```

Comment:
checkin lock file intentional?

Reply:
Yep, for CI/Dev dependency locking. The deps in the pyproject.toml file is where deps are pulled from when installing the package